lhotse/augmentation/lowpass.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import Literal
+
+import numpy as np
+import scipy.signal
+import soundfile as sf
```

Comment:
Can you also move all soundfile imports to local function scopes? IIRC importing this globally used to silently break documentation builds.